### PR TITLE
Create an api to add/get company_profile

### PR DESCRIPTION
**Pull request by meriem26**

### CODE REVIEW

File: server/services/database/companies.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        resolve(result.rows);
+      }
+    });
+  });
+};
+
+const registerCompany = ({ name, description, industry, user_id }) => {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      "INSERT INTO company_profile (name ,description , industry ,user_id) VALUES($1,$2,$3,$4)",
```
Comment (khaledkz) [inline server/services/database/companies.js]: Do you need to add password?

File: server/services/database/companies.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      [name, description, industry, user_id],
+      (error, result) => {
+        console.log("Database", error, result);
```
Comment (islam95) [inline server/services/database/companies.js]: Remove console.log

File: server/services/database/companies.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        if (error) {
+          reject(error);
+        }
+        console.log(result);
```
Comment (islam95) [inline server/services/database/companies.js]: remove console.log

**Comment by islam95** [comment]
You need to fix merge conflicts before I merge this

**Pull request merged**